src/Controls/tests/DeviceTests/Elements/Shell/ShellTests.Windows.cs

```diff
@@ -23,6 +24,46 @@ protected Task CheckFlyoutState(ShellHandler handler, bool desiredState)
 			return Task.CompletedTask;
 		}
 
+		[Theory(DisplayName = "Shell FlyoutBackground Initializes Correctly")]
+		[InlineData("#FF0000")]
+		[InlineData("#00FF00")]
+		[InlineData("#0000FF")]
+		[InlineData("#000000")]
+		public async Task ShellFlyoutBackgroundInitializesCorrectly(string colorHex)
+		{
+			SetupBuilder();
+			
+			var expectedColor = Color.FromArgb(colorHex);
+
+			var shell = await CreateShellAsync((shell) =>
+			{
+				shell.FlyoutBehavior = FlyoutBehavior.Locked;
+				shell.FlyoutBackground = new SolidColorBrush(expectedColor);
+
+				var shellItem = new FlyoutItem();
+				shellItem.Items.Add(new ContentPage());
+				shell.Items.Add(shellItem);
+			});
+
+			await CreateHandlerAndAddToWindow<ShellHandler>(shell, (handler) =>
+			{
+				var rootNavView = handler.PlatformView;
+				var shellItemView = shell.CurrentItem.Handler.PlatformView as MauiNavigationView;
+				var expectedRoot = UI.Xaml.Controls.NavigationViewPaneDisplayMode.Left;
+				var expectedShellItems = UI.Xaml.Controls.NavigationViewPaneDisplayMode.LeftMinimal;
+
+				Assert.Equal(expectedRoot, rootNavView.PaneDisplayMode);
+				Assert.NotNull(shellItemView);
+				Assert.Equal(expectedShellItems, shellItemView.PaneDisplayMode);
+
+				return Task.CompletedTask;
+			});
+
+			await Task.Delay(100);
```

Comment:
PR looks good

Can we change this `await` to use `AssertionExtentions.Wait` and then we'll merge. 

So, I think this regressed because I added code higher up that calls `RefreshThemeResource` on the container window

![image](https://user-images.githubusercontent.com/5375137/218140372-317613b4-acde-48c6-8405-be40b7470823.png)

Which reapplies all the theme resources and wipes out PaneBackground. 

:-/

Reply:
Nice suggestion Shane, updated the test.